bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/CoolItem.java

```diff
@@ -495,7 +495,7 @@ Point getSizeInPixels() {
 	rect.left -= margins.cxLeftWidth;
 	rect.right += margins.cxRightWidth;
 	if (!parent.isLastItemOfRow (index)) {
-		rect.right += (parent.style & SWT.FLAT) == 0 ? CoolBar.SEPARATOR_WIDTH : 0;
+		rect.right += (parent.style & SWT.FLAT) == 0 ? DPIUtil.autoScaleUp(CoolBar.SEPARATOR_WIDTH, getZoom())  : 0;
```

Comment:
This changes the existing behavior and rather seems like a fix for an existing bug than a necessity in terms of multi-zoom support (also see my other detailed comment on changed behavior). I would propose to address such existing bugs and the adaptation for multi-zoom support (of which the latter is the goal of this PR) in different PRs to separate concerns and ease review and discussion.

Reply:
Removed DPIUTil calls for all the places where it wasnt there earlier